meson_options.txt

```diff
@@ -395,4 +395,11 @@ option('ibm-led-extensions',
     value : 'disabled',
     description : '''Enable the IBM LED extensions such as lamp test and system
                      attention indicators.'''
-)
\ No newline at end of file
+)
+
+option(
+    'hw-isolation',
+    type : 'feature',
+    value : 'disabled',
```

Comment:
if you want, you can enable this here. It gives it more CI testing. See https://github.com/ibm-openbmc/bmcweb/pull/924

Reply:
Thank you. I have enabled it